### PR TITLE
Porting cleanup

### PR DESCRIPTION
**Pull request by Nightenom**
# Changes proposed in this pull request:
- removed a lot of unreferenced methods and constants
- transformed few classes to records
- guis are working, rendering needs more proof testing tomorrow

Review please

### CODE REVIEW

File: src/main/java/com/ldtteam/structurize/event/ClientEventSubscriber.java [hunk 1]
```diff
@@ -172,25 +159,7 @@ public static void onClientTickEvent(final ClientTickEvent event)
...
-            Minecraft.getInstance().getProfiler().pop();
-        }
 
         Minecraft.getInstance().getProfiler().pop();
     }
-
-    /**
-     * Used to catch the scroll when no gui is open.
-     *
-     * @param event the catched event.
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/structurize/event/ClientEventSubscriber.java:188]: Don't we need this anymore?

**Comment by Raycoms** [comment]
Looks fine overally. Did you test this with a few different things? (Different schematics, shape tool, scan tool replacement UI, etc)

File: src/main/java/com/ldtteam/structurize/event/ClientEventSubscriber.java (hunk 1 above)
Comment (Nightenom) [reply src/main/java/com/ldtteam/structurize/event/ClientEventSubscriber.java:188]: its in blockui

Comment (Raycoms) [reply src/main/java/com/ldtteam/structurize/event/ClientEventSubscriber.java:188]: ah okay

**Comment by Raycoms** [comment]
Tested and works. I'm not 100% with the border rendering yet. But it's good enough to merge. I believe that might even be on blockUI side.

**Comment by Raycoms** [comment]
Just noticed entity rendering is still broken.

**Comment by Raycoms** [comment]
I will merge this as it fixes a series of other things. I did some debugging and I got closer to why entities render so messed up. 
Doing this: 

```
        RenderSystem.getModelViewStack().pushPose();
        RenderSystem.getModelViewStack().mulPoseMatrix(matrixStack.last().pose());
        RenderSystem.applyModelViewMatrix();
```

before the entity rendering will double apply the viewpoint translations to the entities. Weirdly enough, a bunch of other things still seriously misbehave.

**Comment by Nightenom** [comment]
Well, you know, draft prs are drafts for reason, and also this:
>  rendering needs more proof testing tomorrow

**Comment by Nightenom** [comment]
> Tested and works. I'm not 100% with the border rendering yet. But it's good enough to merge. I believe that might even be on blockUI side.

Well, mojang rendering sucked, what you see is that the box border is now properly scaled while the old one was immune to gui scaling

**Comment by Raycoms** [comment]
So we just have to make the lines smaller by default now

Sent from ProtonMail mobile

-------- Original Message --------

**Pull request merged**